### PR TITLE
Remove obsolete j9sock functions

### PR DESCRIPTION
**Pull request by pshipton**
Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

File: runtime/tests/port/main.c [hunk 1]
```diff
@@ -296,114 +296,110 @@ signalProtectedMain(struct J9PortLibrary *portLibrary, void *arg)
 	}
 
 	rc = 0;
-	if (serverName[0]) {
```
Comment (keithc-ca) [inline runtime/tests/port/main.c]: I think the local variable should also be removed along with the option that sets it.

Comment (pshipton) [reply runtime/tests/port/main.c]: Updated.

**Comment by keithc-ca** [comment]
Jenkins test sanity,extended aix,osx,plinux,xlinux,zlinux,win,win32 jdk8,jdk11

**Comment by keithc-ca** [comment]
The AIX failure is #3081.

**Pull request merged**